.travis.yml

```diff
@@ -1,10 +1,9 @@
+dist: xenial
 language: java
 
 jdk:
-- openjdk8
```

Comment:
Why is jdk8 removed?

Reply:
For some reason, both of these jdk builds fail with `JAVA_HOME not set` error. 
Also see the following error in [build](https://app.travis-ci.com/github/okta/okta-idx-java/jobs/572288491)
![Uploading Screen Shot 2022-06-02 at 3.36.04 PM.png…]()